js/symbol_utils.js

```diff
@@ -0,0 +1,112 @@
+// functions related to the symbol groups
+
+// each pie is a group of pie slices and a pie tin (a circle). here add the pie groups
```

Comment:
so this wouldn't really be decreasing the number of nodes from the current implementation, right? There would still be 3223 circles + 3223 pie groups (with 5 slices each).

Reply:
yeah. i spent a long time drawing on paper to come up with a good plan for transitions, but things got complicated when i started thinking about transitions from one single category to another, and from pies to total, and from single categories to total - those things don't lend themselves as well to the wiper transitions, and so i more or less came to terms with needing a circle in the mix anyway. also, when i tried some wiper transitions they were cripplingly slow.